lib/crud-factory.js

```diff
@@ -20,7 +20,7 @@ module.exports = function(app, resources,actionNames){
     if(name === "create"){
       return !_.isArray(query);
     }else{
-      return _.isString(query) || _.isNumber(query);
+      return (_.isString(query) && query !== '') || _.isNumber(query);
```

Comment:
Not sure, `_.isEmpty` suggests it's an object, while it's only strings acceptable here